### ISSUE TITLE
Live page render includes links to plugin javascript and css

### ISSUE THREAD

**Issue by stevedrobinson** [Webmecanik/plugin-grapesjs-builder#22]
When viewing the final page created by the builder (not logged into Mautic, as an outside visitor), the builder javascript and css are linked from the page html, resulting in errors in the console:
![image](https://user-images.githubusercontent.com/866855/96031928-9d432680-0e23-11eb-88f6-a20009d938d0.png)

**Comment by alexhammerschmied** [comment]
Yes. The finished page holds all links to the builder css and js files. We need to purge this code on save.
